contracts/assemblyscript/nep21-basic/main.ts

```diff
@@ -59,12 +61,13 @@ export function transfer_from(owner_id: AccountId, new_owner_id: AccountId,  amo
     const allowance = allowanceRegistry.getSome(key)
     assert(allowance >= amount, ERR_INSUFFICIENT_ESCROW_BALANCE)
 
-    allowanceRegistry.set(key, u128.sub(allowance, amount))  
+    allowanceRegistry.set(key, u128.sub(allowance, amount))
   }
 
   const balanceOfOwner = balanceRegistry.getSome(owner_id)
+  // please explain the !
```

Comment:
it's a Typescript assertion for non-null values bc `get` can return null but I'm sure it won't in this case since I'm providing a default

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator